v6.19.4 (#12015)
mui/mui-x: CHANGELOG.md

All notable changes to this project will be documented in this file.
See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.

## v6.19.4

_Feb 9, 2024_

> We'd like to offer a big thanks to the 4 contributors who made this release possible. Here are some highlights ✨:

Reviewer: ```suggestion
We'd like to offer a big thanks to the 9 contributors who made this release possible. Here are some highlights ✨:
```

> We'd like to offer a big thanks to the 9 contributors who made this release possible. Here are some highlights ✨:

Reviewer: Sorry for miscalculating initially. 🙈 
```suggestion
We'd like to offer a big thanks to the 10 contributors who made this release possible. Here are some highlights ✨:
```

> - 🌍 Improve  Danish (da-DK) locale on the Data Grid (#11972) @ShahrazH

Reviewer: ```suggestion
- 🌍 Improve Danish (da-DK) locale on the Data Grid (#11972) @ShahrazH
```

> - 🌍 Improve  Danish (da-DK) locale on the Data Grid (#11831) @danielmishan85

Reviewer: ```suggestion
- 🌍 Improve  Danish (da-DK) locale on the Data Grid (#11972) @ShahrazH
```

- [DataGrid] Fix row reorder with cell selection (#11878) @PEsteves8
- [DataGrid] Replace `eval` with `new Function` (#11962) @cherniavskii
- [l10n] Improve Danish (da-DK) locale (#11972) @ShahrazH

#### `@mui/x-data-grid-pro@v6.19.4` [![pro](https://mui.com/r/x-pro-svg)](https://mui.com/r/x-pro-svg-link 'Pro plan')

Same changes as in `@mui/x-data-grid@v6.19.4`.

#### `@mui/x-data-grid-premium@v6.19.4` [![premium](https://mui.com/r/x-premium-svg)](https://mui.com/r/x-premium-svg-link 'Premium plan')

Same changes as in `@mui/x-data-grid-pro@v6.19.4`, plus:

- [DataGridPremium] Fix autosize grouping cell (#11990) @romgrk
- [DataGridPremium] Fix error after closing print export (#11889) @cherniavskii

### Date Pickers

#### `@mui/x-date-pickers@v6.19.4`

- [pickers] Avoid relying on locale in Luxon `isWithinRange` method (#11940) @LukasTy

#### `@mui/x-date-pickers-pro@v6.19.4` [![pro](https://mui.com/r/x-pro-svg)](https://mui.com/r/x-pro-svg-link 'Pro plan')

Same changes as in `@mui/x-date-pickers@v6.19.4`.

### Charts / `@mui/x-charts@v6.19.4`

- [charts] Add `reverse` property to axes (#11959) @alexfauquette

> - [charts] Allows series ids to be numbers (#11960) @alexfauquette

Reviewer 2: ```suggestion
- [charts] Allow series ids to be numbers (#11960) @alexfauquette
```

> - [charts] Fix Proptypes by support string values for axes (#11953) @alexfauquette

Reviewer 2: ```suggestion
- [charts] Fix Proptypes error by supporting string values for axes (#11953) @alexfauquette
```

### Docs

- [docs] Add a note about `AdapterDateFnsV3` on the Getting Started page (@flaviendelangle) (#11987) @github-actions[bot]
- [docs] Avoid the use of MUI Core @oliviertassinari
- [docs] Fix API links (#11930) @alexfauquette
- [docs] Fix `ChartsTooltip` typo (@thisisharsh7) (#11967) @github-actions[bot]

> - [docs] Refactor `Localization` documentation sections (@LukasTy) (#11997) @github-actions[bot]

Reviewer: ```suggestion
- [docs] Refactor `Localization` documentation sections (#11997) @LukasTy
```

> - [code] Simplify bug reproduction (#11849) (#11932) @alexfauquette

Reviewer: ```suggestion
- [code] Simplify bug reproduction (#11932) @alexfauquette
```
